plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -233,7 +233,7 @@ class HorizontalPodAutoscaler < Base
       private
 
       def client
-        @deploy_group.kubernetes_cluster.autoscaling_client
+        @deploy_group.kubernetes_cluster.client('autoscaling/v1')
```

Comment:
Maybe worth putting these types (i.e. 'autoscaling/v1')  in a constant hash in case they change? Then use symbols for all types?

Reply:
apis are backwards compatible, so they should not change ... someone might need new apis, but that does not mean we should change everyone who used v1 before